pallets/disputes/src/tests.rs

```diff
@@ -182,6 +182,33 @@ fn vote_on_dispute_assert_last_event() {
     });
 }
 
+
+#[test]
+fn vote_on_dispute_assert_last_event_on_initialize() {
+    new_test_ext().execute_with(|| {
+        let dispute_key = 10;
+        let jury = get_jury::<Test>(vec![*CHARLIE, *BOB]);
+        let specifics = get_specifics::<Test>(vec![0, 1]);
+        assert_ok!(<PalletDisputes as DisputeRaiser<AccountId>>::raise_dispute(
+            dispute_key,
+            *ALICE,
+            jury,
+            specifics,
+        ));
+        assert_ok!(PalletDisputes::vote_on_dispute(
+            RuntimeOrigin::signed(*BOB),
+            dispute_key,
+            true
+        ));
+        ///trying to expire the timelimit for the given dispute
+        run_to_block::<Test>(11);
```

Comment:
Please if you can, dont use static values like this. 
If we change  `VotingTimeLimit` we want this `11` value to change with it.
```suggestion
       let current_block = frame_system::Pallet::<Test>::current_block_number();
       run_to_block::<Test>(current_block_number + <Test as Config>::VotingTimeLimit::get());
```

Reply:
Done